Lib/idlelib/help.html

```diff
@@ -364,11 +364,18 @@ <h3>Help menu (Shell and Editor)<a class="headerlink" href="#help-menu-shell-and
 <dt>Clear Breakpoint</dt>
 <dd>Clear the breakpoint on that line.</dd>
 </dl>
-<p>Shell and Output windows have the following.</p>
+<p>Shell and Output windows also have the following.</p>
 <dl class="docutils">
 <dt>Go to file/line</dt>
 <dd>Same as in Debug menu.</dd>
 </dl>
+<p>The Shell window also has an output squeezing facility explained in the
+the <em>Python Shell window</em> subsection below.</p>
+<dl class="docutils">
+<dt>Squeeze</dt>
+<dd>If the cursor is over an output line, squeeze all the output between
+the code above and the prompt below down to a ‘Squeezed text’ button.</dd>
```

Comment:
```suggestion
the code above and the prompt below down to a ‘Squeezed text’ label.</dd>
```

Reply:
This looks like it would work great, except I just realized that IDLE doc changes have to be made in parallel in both the .rst and .html files.  That is easiest to do by changing the .rst on my machine and then generating the revised .html with Sphinx.